vm-allocator/src/system.rs

```diff
@@ -119,7 +119,7 @@ impl SystemAllocator {
         self.platform_mmio_address_space.allocate(
             address,
             size,
-            Some(align_size.unwrap_or(get_page_size())),
+            Some(align_size.unwrap_or_else(get_page_size)),
```

Comment:
why is this and below changed to `unwrap_or_else`? Page size shouldn't change should it?

Reply:
rust 1.65 and later has clippy issue with unwrap_or.